src/sentry/middleware/stats.py

```diff
@@ -28,7 +29,7 @@ def process_exception(self, request, exception):
 
 class RequestTimingMiddleware(object):
     allowed_methods = ("POST", "GET")
-    allowed_paths = ("sentry.web.api", "sentry.api.endpoints")  # Store endpoints
+    allowed_paths = getattr(settings, "SENTRY_REQUEST_METRIC_ALLOWED_PATHS", ("sentry.web.api", "sentry.api.endpoints"))  # Store endpoints
```

Comment:
Not sure, but maybe it makes sense for the default to just be an empty tuple?

Reply:
I thought about that.  I wanted to be on the safe side and made sure that sentry continue to send data even if we messed up the config.